nextjs-13/src/stylesheets/components/_list-newsletter.scss

```diff
@@ -0,0 +1,23 @@
+.list-newsletter {
+  font-weight: var(--font-weight-5);
+  width: 500px;
+
+  ul {
+    list-style-type: none;
+    padding-left: 0;
+    margin-top: var(--size-4);
+
+    li {
+      display: flex;
+      flex-direction: row;
+      background: var(--gray-0);
+      border-radius: var(--radius-conditional-2);
+      padding: var(--size-2);
+      color: black;
+      cursor: pointer;
+      --_size: var(--font-size-1);
+      margin-bottom: var(--size-3);
+      box-shadow: var(--shadow-2);
```

Comment:
There is room to improve the ordering of the styles for readability:

```css
    li {
     // always place variables at the top of the block
      --_size: var(--font-size-1); 

      // place any style that can affect the positioning first
      display: flex;
      flex-direction: row;
      margin-bottom: var(--size-3);
      padding: var(--size-2);

      // font related styles nexts
      color: black;
      
      // the rest
      cursor: pointer;
      background: var(--gray-0);
      border-radius: var(--radius-conditional-2);
      box-shadow: var(--shadow-2);
   }

```

Reply:
Thank you a lot for these suggestions about the CSS! To be honest, I don't help much experience with the frontend(so the CSS too), so please keep assisting me on my CSS improvement journey 🤣

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/110/commits/982cdae493d9ee69a7091072c4dc136019a67cd7#diff-099e542ab2be0b36ba6ee7666ef7e5ef285c13f7a6e3e70bb3c854526d41b0e1R11)